src/components/user-chip/UserChip.tsx

```
@@ -30,22 +30,28 @@
   users,
 }: UserChipProps) {
-  return isMulti && !!users?.length ? (
-    <div className={classes.userChipContainer}>
-      <div className={classes.userChipImageContainer}>
-        <img
-          src={users[0].profile_img_url ? users[0].profile_img_url : ''}
-          className={clsx(classes.profileImage, classes.firstImage)}
-        />
-        {users?.length > 1 && (
+  return isMulti ? (
+    !!users?.length ? (
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)